proto/api/v1/compliance_service.proto

```diff
@@ -70,6 +72,12 @@ message GetComplianceStandardsResponse {
     repeated ComplianceStandardMetadata standards = 1;
 }
 
+message UpdateComplianceRequest {
+
+    string id = 1;
+    bool hidden = 2;
```

Comment:
I like the var name you used in the API `hideScanResults`. Recommend making the two field names consistent by changing this to `hideScanResults`.

Reply:
api spec will be exposed to customers so it might be better to use shorter name

Reply:
A clear name over a short name wins anytime :)